corehq/apps/domain/forms.py

```diff
@@ -641,6 +659,9 @@ def _save_release_mode_setting(self, domain):
             setting_obj.is_visible = self.cleaned_data.get("release_mode_visibility")
             setting_obj.save()
 
+    def _save_orphan_case_alerts_setting(self, domain):
+        domain.orphan_case_alerts_warning = self.cleaned_data.get("orphan_case_alerts_warning")
```

Comment:
Nit
```suggestion
        domain.orphan_case_alerts_warning = self.cleaned_data.get("orphan_case_alerts_warning", False)
```

Adding the `False` will make sure it's explicitly `False` and not just falsy as in the case of `None`. Your choice if you want to update.

Reply:
Updated